FRAuth/FRAuth/DeviceBinding/UserKeySelector.swift

```diff
@@ -0,0 +1,58 @@
+// 
+//  UserKeySelector.swift
+//  FRAuth
+//
+//  Copyright (c) 2022-2023 ForgeRock. All rights reserved.
+//
+//  This software may be modified and distributed under the terms
+//  of the MIT license. See the LICENSE file for details.
+//
+
+import UIKit
+
+/// Protocol for selecting ``UserKey``
+public protocol UserKeySelector: AnyObject {
+    /// Method to choose from multiple keys
+    /// - Parameter userKeys: List of ``UserKey``s
+    /// - Parameter selectionCallback: ``UserKey`` selection callback
+    func selectUserKey(userKeys: [UserKey], selectionCallback: @escaping UserKeySelectorCallback)
+}
+
+/// Default implementation for UserKeySelector protocol
+public class DefaultUserKeySelector: NSObject, UserKeySelector {
+    
+    /// Default implementation with a simple action sheet UI
+    public func selectUserKey(userKeys: [UserKey], selectionCallback: @escaping UserKeySelectorCallback) {
+        DispatchQueue.main.async {
+            let keyWindow = UIApplication.shared.windows.filter {$0.isKeyWindow}.first
+            var topVC = keyWindow?.rootViewController
+            while let presentedViewController = topVC?.presentedViewController {
+                topVC = presentedViewController
+            }
+            guard let topVC = topVC else { return }
+            
+            let actionSheet = UIAlertController(title: "Select User", message: nil, preferredStyle: .actionSheet)
```

Comment:
Can we make the Alert values either localizable or passable from the method, so that we enable developers to provide localized strings?